public/Aryan Suri/Licenses/licenses.js

```diff
@@ -0,0 +1,255 @@
+
+					break;
+				case "cc-publicdomain":
+					return null;
+					break;
+			}
+
+		} else {
+			$(modalB).show();
+			modalC.innerHTML = `<span> The license of the content on this page is unselected. Please review the Contributors and Attributions section or the content author(s) for clarification of the applicable license(s). </span>`
+
+		}
+	}
+
+
+	function ccPageLabel() {
+		const cc = getCC();
+		if (cc) {
+			const pageLabel = document.createElement("li");
+			pageLabel.classList.add("pageInfo");
+			if (cc.label === "gnu") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/gpl.png" alt="GNU General Public License"/></a>`;
+			}
+			else if (cc.label === "gnudsl") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><p style="color: black; font-size: small">GNU Design Science License</p></a>`;
+			}
+			else if (cc.label === "gnufdl") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/gfdl.png" alt="GNU Free Documentation License"/></a>`;
+			}
+			else if (cc.label === "arr") {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><img style="height: 30px" src="https://awesomefiles.libretexts.org/Social Share Kit/arr.png" alt="All Rights Reserved"/></a>`;
+			}
+			else {
+				pageLabel.innerHTML = `<span> <a style="width: max-content; width: -moz-max-content; overflow: initial; font-size:30px" href="${cc.link}"><i class='cc ${cc.label}'></i></a>`;
+			}
+
+
+			if ($("li.elm-last-modified").length)
+				$("li.elm-last-modified")[0].before(pageLabel);
+			else
+				pageLabel.style.display = "none";
+		}
+		const pageNumberHolder = document.getElementById("pageNumberHolder");
+		if ($("li.elm-page-restriction").length)
+			$("li.elm-page-restriction")[0].after(pageNumberHolder);
+		else if (window.location.host.startsWith('query') && $("li.elm-last-modified").length)
+			$("li.elm-last-modified")[0].after(pageNumberHolder);
+
+		else
+			pageNumberHolder.style.display = "none";
+
+		const isAdmin = document.getElementById("adminHolder").innerText === 'true';
+		if ($(".elm-social-share").length) {
+			let html = '<div class="ssk-group optimize"><div href="" class="ssk ssk-facebook"></div><div href="" class="ssk ssk-twitter"></div><div id="batchPrint"></div>';
+			if (!isAdmin)
+				html += '<a href="https://donorbox.org/libretexts" class="custom-dbox-popup notSS" id="donate"><span>Donate</span></a></div>';
+			$(".elm-social-share")[0].innerHTML = html;
+			SocialShareKit.init();
```

Comment:
Man this code is old
```suggestion
			if (!isAdmin)
				html += '<a href="https://donorbox.org/libretexts" class="custom-dbox-popup notSS" id="donate"><span>Donate</span></a></div>';
			else
			        html +='</div>';
			$(".elm-social-share")[0].innerHTML = html;
			//SocialShareKit.init();
```